Reduce the volume of the death sound on clock work mobs

Garen7 opened the pull request:
This will affect all code/game/gamemodes/clock_cult/clock_mobs which at this moment is only marauders and the eminence.

[The old and new sound files.zip](https://github.com/tgstation/tgstation/files/1629990/The.old.and.new.sound.files.zip)

[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: 
tweak: Reduce clock work mob death sound file's volume to 80%
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

I made these changes at the request of a player who said that when dying as a marauder the sound is a little too loud. I only reduced it to 80% because it seems important for other players to know when clock mobs die. I edited the .oog since I did not see a way in the code to lower the volume.

Garen7 committed:
Reduce the volume of the death sound on clock work mobs

I edited the .oog since I did not see a way in the code to lower the volume. This will affect all code/game/gamemodes/clock_cult/clock_mobs which at this moment is only maraders and the eminence.

Potato-Masher commented:
Good, they were kinda loud for anyone standing nearbye. Sounds like an okay change.

ShizCalev reviewed (changes requested):
Adjust the volume for `/datum/species/golem/clockwork/spec_death`, `/obj/item/toy/plush/plushvar/proc/clash_of_the_plushies`, & `/obj/structure/destructible/clockwork/massive/ratvar/proc/clash_of_the_titans`

Garen7 committed:
Non-clockwork mobs are no longer affected

adjusted volume in the code so that the clockwork golems, ratvar, and ratvar plushies are unaffected by the audio file changes
